deployability/modules/workflow_engine/tests/data/wf-ko-no-path-on-cleanup.yml

```diff
@@ -0,0 +1,169 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+version: 0.1
+description: This workflow is used to test agents deployment with a single manager.
+variables:
+  agents-os:
+    - linux-ubuntu-22.04-amd64
+  manager-os: linux-ubuntu-22.04-amd64
+  infra-provider: vagrant
+  working-dir: /tmp/dtt1
+
+tasks:
+  # Generic agent test task
+  - task: "run-agent-tests-{agent}"
+    description: "Run tests uninstall for the {agent} agent."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/testing/main.py
+          - inventory: "{working-dir}/agent-{agent}/inventory.yaml"
+          - dependencies:
+            - manager: "{working-dir}/manager-{manager-os}/inventory.yaml"
+            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
+          - tests: "install,register,stop"
+          - component: "agent"
+          - wazuh-version: "4.7.1"
+          - wazuh-revision: "40709"
```

Comment:
Shouldn't the version and revision be parametrized?

Reply:
The file `deployability/modules/workflow_engine/tests/data/wf-ko-no-path-on-cleanup.yml` is used by the test `test_schema_validator.py::test_preprocess_data_ko`. The file content is validated against the schema using the SchemaValidator class in this test. The preprocess data does not validate the `wazuh-version` and `wazuh-revision` fields, so making it parametrizable will not affect the test unless the tested function is modified.